wolox-training/ap-swift
# Book detail View for Wishlist, Library and Rentals

Books on BookInfoTableViewCell

## Summary ##
**Trello card**: https://trello.com/c/OByrbt9V/27-vista-de-detalle-de-un-libro-desde-libros-alquilados
**Trello card**: https://trello.com/c/49U8fdI6/24-vista-de-detalle-de-un-libro-desde-wishlist

Book detail view from Wishlist, Library and Rentals.
Show correct labels and buttons for Available, Not Available and Rented books on the Book Details View

## Test Plan ##

 * Open the app
  * Login with Google
  * The main tab bar view should appear after a successfull login.
  * Press on Rentals tab
  * My Rentals tab should appear and load book cells
  * Press on a cell
  * Book detail view should appear with bookInfo for a Rented book
  * Press on Wishes tab
  * Wishes tab should appear and load book cells
  * Press on a cell
  * Book detail view should appear with bookInfo for a wish book

## Screenshots ##
### iPhone 7

![details-rentals](https://cloud.githubusercontent.com/assets/25587882/24121140/ed6ef2f2-0d95-11e7-84a7-f7312807955a.png)

![details-library](https://cloud.githubusercontent.com/assets/25587882/24200212/295afaee-0eeb-11e7-913e-bca7fa65f8ac.png)


## Screencast ##

![rental-detail](https://cloud.githubusercontent.com/assets/25587882/24200279/702f30f2-0eeb-11e7-81d6-9922512b77a6.gif)

## Known issues ##

The API is not working for Returning Books and Removing them from the Whislist. Also there is no available book on the api, all the books are rented
